### DISCUSSION TITLE
Change MinValue/MaxValue to reasonable values on all rides

### DISCUSSION THREAD

**Discussion by spacek531** [OpenRCT2/OpenRCT2#21753] [Feature requests] [answered]
When hacking a ride to use powered launch, most rides also require 'Unlock Operating Limits' to be enabled, because the RTD's OperatingSettings' MaxValue property is zeroed. It would be nice if all ride types had usable ranges for powered launch, without requiring an additional cheat enabled.

Because OperatingSettings' MinValue and MaxValue represent several different properties (e.g. Dodgems runtime) some investigation must be taken to determine all the affected operating settings these properties affect.

**Comment by MarcelVos96** [comment]
I think it's best to do this ride by ride and compare it to other ride types that already have access to boosters.

For example, the mine train coaster could behave the same as the wooden coaster, the wild mice coasters could be the same as the junior coaster, the inverted coaster could be the same as the twister coaster, and so on.

**Comment by spacek531** [comment, marked as answer]
Implemented in https://github.com/OpenRCT2/OpenRCT2/pull/25994
